docs/en/observability/logs-troubleshooting.asciidoc

```diff
@@ -1,11 +1,20 @@
 [[logs-troubleshooting]]
 = Troubleshoot logs
 
+Use this page to find possible solutions for errors your encountering with your logs. This troubleshooting page is broken into the following sections:
```

Comment:
```suggestion
Use this page to find possible solutions for errors your encountering with your logs. This troubleshooting page is divided into the following sections:
```
Just a thought: For non-native English speakers the word "broken" might be a little jarring here.